lms/models/user.py

```diff
@@ -32,7 +32,7 @@ class User(CreatedUpdatedMixin, Base):
     )
     application_instance = sa.orm.relationship("ApplicationInstance")
 
-    user_id = sa.Column(sa.Unicode, nullable=False)
+    user_id: Mapped[str] = mapped_column(sa.Unicode, nullable=False)
```

Comment:
Adopt the new style to make the typechecker happy. 

Eventually we could do a one time refactor to change all the models to the new style.